### PR TITLE
Improvements for the equations plugin

### PR DESCRIPTION
**Pull request by kulla**
### Included in this PR:

* Make explanation field focusable with "tab" key
* Omit sign field with tab
* Add button "add new equation"
* Start with two lines of equations
* Fix bug with missalignment and \underbrace and \overbrace
* Variable length down arrow

### TODOS:

- [x]  Add "Add new equation" button

### CODE REVIEW

File: packages/private/edtr-io/src/plugins/equations/editor.tsx [hunk 1]
```diff
@@ -411,40 +374,30 @@ function InlineMath(props: InlineMathProps) {
...
+      onMoveOutLeft={onFocusPrevious}
     />
   )
 }
 
 type GridFocusState = {
   row: number
   column: number
-} | null
+}
```
Comment (inyono) [inline packages/private/edtr-io/src/plugins/equations/editor.tsx:395]: Should be an `interface`.

File: packages/private/edtr-io/src/plugins/equations/editor.tsx [hunk 2]
```diff
@@ -411,40 +374,30 @@ function InlineMath(props: InlineMathProps) {
...
 
 interface GridFocus {
-  focus: GridFocusState
-  isFocused: (payload: { row: number; column: number }) => boolean
-  setFocus: (focus: GridFocusState) => void
+  focus: GridFocusState | null
+  isFocused: (cell: GridFocusState) => boolean
+  setFocus: (cell: GridFocusState) => void
```
Comment (inyono) [inline packages/private/edtr-io/src/plugins/equations/editor.tsx:400]: This worked with `null` previously (to clear the focus). Don't we need anymore?

File: packages/private/edtr-io/src/plugins/equations/renderer.tsx [hunk 3]
```diff
@@ -110,3 +111,51 @@ export function EquationsRenderer({ state }: EquationsProps) {
...
+
+  return (
+    <td
+      style={{
+        backgroundImage: `url('data:image/svg+xml;base64,${btoa(downArrow)}')`,
+        backgroundSize: '20px calc(100% - 10px)',
+        backgroundRepeat: 'no-repeat',
+        backgroundPosition: 'center 5px',
+      }}
+    ></td>
```
Comment (inyono) [inline packages/private/edtr-io/src/plugins/equations/renderer.tsx:159]: self-closing tag

File: packages/private/edtr-io/src/plugins/equations/editor.tsx (hunk 2 above)
Comment (kulla) [reply packages/private/edtr-io/src/plugins/equations/editor.tsx:400]: In the current implementation: No. When I remember it well it wasn't also needed when I started to code on this issue

**Comment by kulla** [comment]
@inyono I have got positive feedback. After merging https://github.com/serlo/frontend/pull/1262 this PR can also be merged.

**Pull request merged**